tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -26,7 +26,9 @@ import {
 } from '../../../src/util/logger';
 
 const defaultChromeFlags = ChromeLauncher.defaultFlags()
-  .filter((flag) => flag !== '--disable-extensions');
+  .filter((flag) => {
+    return flag !== '--disable-extensions' && flag !== '--mute-audio';
+  });
```

Comment:
I've been briefly discussing the related issue with @Rob--W yesterday and we agreed that, to avoid "surprises" when we upgrade the chromium-launcher dependency, it may be good to add an additional test case that fails if chromium-launcher adds additional default flags.

You can get the list of the currently expected default flags here: https://github.com/GoogleChrome/chrome-launcher/blob/master/src/flags.ts

would you mind to add an additional test case that compares this "filtered out array of default flags" with a list of expected ones?

Reply:
I think the link you shared is from `12.0`, while the version `web-ext` currently uses is `11.2` – There's at least 4 extra flags between the two versions.

I've pulled the expected flags from [`11.2/flags.ts`](https://github.com/GoogleChrome/chrome-launcher/blob/v0.11.2/src/flags.ts) for now. Let me know if this is okay 👍

Reply:
:+1: yep, you are right, I forgot that we didn't merged #1745 yet and so we are still on the 11.2 release.